library/std/src/diagnostics.qs

```diff
@@ -69,11 +69,45 @@ namespace Microsoft.Quantum.Diagnostics {
         body intrinsic;
     }
 
+    /// # Summary
+    /// Checks whether a qubit is in the |0⟩ state (the ground state), returning true if it is.
```

Comment:
I don't think we're using "ground state" anywhere else to refer to the |0⟩ state, so I would not bring this in this doc. If somebody's writing Q#, they most likely know what |0⟩ is; and "ground state" is the kind of terminology I'd expect to see in chemistry apps, not in the basic concepts

Reply:
I'll just remove it then, it's simpler that way.